spec.md

```diff
@@ -427,21 +427,19 @@ how Platforms might expose these values to their users.
 
 | Response Field | Type | Description |
 | --- | --- | --- |
-| [create](#json-schema-object) | object | The schema definition for creating a Service Instance. |
-| [update](#json-schema-object) | object | The schema definition for updating a Service Instance. |
+| create | [Request/ResponseSchemaObject](#request-response-json-schema-object) | The Service Broker schema definition for creating a Service Instance and the associated response. |
```

Comment:
The "Request/ResponseSchemaObject" link text is slightly inconsistent with the rest of the document:
- there should be no forward slash in the name (all other links use the actual type name from the OpenAPI spec, not the "human-focused" name)
- there should be no "Object" suffix in the link text

Reply:
I like it. Thanks for the feedback. Updated.